### PR TITLE
[Translation] Hindi

### PR DESCRIPTION
**Pull request by durgeshs**

### CODE REVIEW

File: config/translations/hi/create.nodes.yml [hunk 1]
```diff
@@ -1,28 +1,28 @@
-description: 'Create dummy nodes for your Drupal 8 application.'
-help: 'The <info>create:nodes</info> command helps you create dummy nodes.'
-welcome: 'Welcome to the Drupal nodes generator'
+description: 'अपने द्रुपल 8 एप्लीकेशन के लिए डमी नोड्स बनाएँ।'
+help: '<info>create:nodes</info> कमांड आपको डमी नोड्स को बनाने में मदद करता है।'
+welcome: 'द्रुपल नोड्स जनरेटर में आपका स्वागत है।'
```
Comment (enzolutions) [inline config/translations/hi/create.nodes.yml]: Please don't translate Drupal , PHP or others terms

File: config/translations/hi/database.dump.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-description: 'Dump structure and contents of MySQL databases and tables'
+description: 'डंप संरचना और मायसीक्यूल का कंटेंट डेटाबेसेस और टेबल्स'
```
Comment (enzolutions) [inline config/translations/hi/database.dump.yml]: Don't translate MySQL

File: config/translations/hi/database.log.clear.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-description: 'Remove events from DBLog table, filters are available'
+description: 'डीबीलॉग टेबल से इवेंट्स निकालें, फिल्टर उपलब्ध हैं'
```
Comment (enzolutions) [inline config/translations/hi/database.log.clear.yml]: Don't translate DBLog

File: config/translations/hi/database.log.debug.yml [hunk 4]
```diff
@@ -1,19 +1,19 @@
-description: 'Display current log events for the application'
+description: 'आवेदन के लिए वर्तमान लॉग इवेंट्स को प्रदर्शित करे'
 arguments:
-    event-id: 'DBLog event ID'
+    event-id: 'डीबीलॉग इवेंट आईडी'
```
Comment (enzolutions) [inline config/translations/hi/database.log.debug.yml]: Don't translate DBLog

File: config/translations/hi/database.log.debug.yml [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
-    invalid-severity: 'Severity type is invalid, filter was ignored'
-    not-found: 'DBLog event %s wasn''t found'
+    event-id: 'इवेंट आईडी'
+    type: टाइप
+    date: दिनांक
+    message: मैसेज
+    user: यूजर
+    severity: सेवेरिटी
+    invalid-severity: 'सेवेरिटी टाइप अमान्य है, फिल्टर को नजरअंदाज कर दिया गया था'
+    not-found: 'डीबीलॉग इवेंट %s नहीं मिला था'
```
Comment (enzolutions) [inline config/translations/hi/database.log.debug.yml]: Don't translate DBLog

File: config/translations/hi/database.restore.yml [hunk 6]
```diff
@@ -1,7 +1,7 @@
-description: 'Restore structure and contents of MySQL databases and tables'
+description: 'माय स्क्ल डाटाबेस और टेबल्स ले कंटेंट और संरचना को रिस्टोर करे'
```
Comment (enzolutions) [inline config/translations/hi/database.restore.yml]: Don't translate MySQL

File: config/translations/hi/database.table.debug.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
...
-help: 'The <info>database:table:debug</info> command helps you debug database tables.'
+description: 'एक दिए गए डेटाबेस में सभी टेबल्स दिखाएँ.'
+help: '<info>database:table:debug</info> कमांड आप को डेटाबेस टेबल्स डिबग करने में मदद करता है।'
 arguments:
-    database: 'Database key from settings.php'
+    database: 'Settings.php से डाटाबेस कुंज'
 messages:
-    table-show: 'Showing tables for %s database'
-    table: Table
+    table-show: '%S डेटाबेस के लिए टेबल दिखाओ'
```
Comment (enzolutions) [inline config/translations/hi/database.table.debug.yml]: Proper keyword is %s

File: config/translations/hi/database.connect.yml [hunk 8]
```diff
@@ -1,8 +1,8 @@
-description: 'Launch a DB client if it''s available'
+description: 'लॉन्च एक डीबी क्लाइंट अगर यह उपलब्ध है'
```
Comment (enzolutions) [inline config/translations/hi/database.connect.yml]: Don't translate DB

**Comment by durgeshs** [comment]
Done :)

**Comment by enzolutions** [comment]
Thanks @durgeshs

**Pull request merged**